presto-native-execution/src/test/java/com/facebook/presto/spark/AbstractTestPrestoSparkQueries.java

```diff
@@ -95,4 +100,17 @@ protected Map<String, String> getNativeExecutionShuffleConfigs()
         sparkConfigs.put(FALLBACK_SPARK_SHUFFLE_MANAGER, "org.apache.spark.shuffle.sort.SortShuffleManager");
         return sparkConfigs.build();
     }
+
+    protected ImmutableList<Module> getNativeExecutionModules()
```

Comment:
It would be better to move this function down into `PrestoSparkNativeQueryRunner` which creates/handles Native specific behavior.

Reply:
The idea is to make the`PrestoSparkNativeQueryRunner` as flexible as possible to take different configs (already taking these) or native modules (which this PR is adding) so different test suites can configure these things depending on their use cases. That been said, the `AbstractTestPrestoSparkQueries` is the one to configure the native behaviors for OSS env, we'll have other test suites (say `AbstractTestPrestoFacebookSparkQueries`) to configure different native behaviors. Does this make make sense?

Reply:
Thanks for clarifying the intent. Makes sense